.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Release Notes:
+> Provide a concise summary of changes for the release notes. Include any new features, bug fixes, or other significant updates. Keep it brief and to the point.
+
+## Checklist:
+> Ensure the following tasks are completed before submission.
+- [ ] My code follows the project style guidelines
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have added tests that prove my fix is effective or my feature works
+- [ ] New and existing tests pass locally with my changes
+- [ ] I have used a code formatting tool to ensure consistent style
+- [ ] I have run a linter to check for code quality issues
```

Comment:
Great idea of having this template. Just add a period at the end of each sentence.

Reply:
Fixed in commit 03dcec8f097eeadc82b4726df46404de52d3be47.